### PR TITLE
Implemented new button with dialog

### LINKED ISSUE #90: New App-view button

### PR DESCRIPTION
**Pull request by Aagaard00**

### CODE REVIEW

File: src/UICrafter.Client/Components/Dialogs/CreateNewDialog.razor [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <MudButton OnClick="() => MudDialog.Cancel()">No</MudButton>
+    </DialogActions>
+</MudDialog>
+
+
+@code {
+    [CascadingParameter]
+    private MudDialogInstance MudDialog { get; set; } = null!;
+
+    protected override async Task OnInitializedAsync()
```
Comment (Nephim) [inline src/UICrafter.Client/Components/Dialogs/CreateNewDialog.razor]: dont use async version

File: src/UICrafter.Client/UICrafter.Client.csproj [hunk 2]
```diff
@@ -23,4 +23,10 @@
     <ProjectReference Include="..\UICrafter.Core\UICrafter.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (Nephim) [inline src/UICrafter.Client/UICrafter.Client.csproj]: Remove this

**Review by Gormiee — APPROVED** [review]
LGTM

**Review by Nephim — APPROVED** [review]
Fix these then LGTM

**Review by Nephim — APPROVED** [review]

**Pull request merged**